website/docs/r/synthetics_monitor_step.html.markdown

```diff
@@ -0,0 +1,100 @@
+---
+layout: "newrelic"
+page_title: "New Relic: newrelic_synthetics_step_monitor"
+sidebar_current: "docs-newrelic-resource-synthetics-step-monitor"
+description: |-
+Create and manage a Synthetics monitor in New Relic.
+---
+
+# Resource: newrelic\_synthetics\_step\_monitor
+
+Use this resource to create, update, and delete the synthetics step monitor in New Relic.
+
+-> **NOTE:** The [newrelic_synthetics_private_location](newrelic_synthetics_private_location.html) resource private minion can take upto 10 minutes to be available through Terraform.
```

Comment:
Take a look at the Additional Examples for the dashboard resource [https://registry.terraform.io/providers/newrelic/newrelic/latest/docs/resources/one_dashboard#additional-examples](https://registry.terraform.io/providers/newrelic/newrelic/latest/docs/resources/one_dashboard#additional-examples)

Let's add something like that for a monitor with a private location and move this note there